sputnikdao2/tests-ava/README.md

```diff
@@ -17,4 +17,8 @@ To run only one test file:
 To run only one test:
 
     npm run test -- -m "root sets*" # matches tests with titles starting with "root sets"
-    yarn test -m "root sets*"       # same thing using yarn instead of npm, see https://yarnpkg.com/
\ No newline at end of file
+    yarn test -m "root sets*"       # same thing using yarn instead of npm, see https://yarnpkg.com/
+
+If debugging:
+
+    NEAR_WORKSPACES_DEBUG=true npm run test "**/upgrade*"
```

Comment:
`"**/upgrade*"` is out of context. I think simply `NEAR_WORKSPACES_DEBUG=true npm run test` should be enough.